packages/landingpage-new/pages/home/component-sample/component-sample.tsx

```diff
@@ -0,0 +1,201 @@
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr Lorem ipsum
+          dolor sit amet, consetetur sadipscing elitr
+        </p>
+      </div>
+      <div className={styles.container}>
+        <div className={styles.chip}>
+          <InoCard disableElevation={true}>
+            <div className={styles.content} slot="content">
+              {showChip && (
+                <InoChip
+                  removable={true}
+                  onChipRemoved={() => {
+                    setChip(false);
+
+                    setTimeout(() => {
+                      setChip(true);
+                    }, 5000);
+                  }}
+                >
+                  Removable Chip
+                </InoChip>
+              )}
+            </div>
+            <div slot="footer" className={styles.footer}>
+              <div className={styles.component}>Chip</div>
+              <div className={styles.label}>Button</div>
+            </div>
+          </InoCard>
+        </div>
+        <div className={styles.slider}>
+          <InoCard disableElevation={true}>
+            <div className={styles.content} slot="content">
+              <div className={styles.range}>
+                <InoRange
+                  min={0}
+                  max={100}
+                  value={rangeValue}
+                  step={1}
+                  onValueChange={(value) => setRangeValue(value.detail)}
+                ></InoRange>
+              </div>
+            </div>
+            <div slot="footer" className={styles.footer}>
+              <div className={styles.component}>Slider</div>
+              <div className={styles.label}>Input</div>
+            </div>
+          </InoCard>
+        </div>
+        <div className={styles.snackbar}>
+          <InoCard disableElevation={true}>
+            <div className={styles.content} slot="content">
+              <InoButton
+                onClick={() => {
+                  setSnackbar(true);
+
+                  setTimeout(() => {
+                    setSnackbar(false);
+                  }, 5000);
+                }}
+              >
+                Show Snackbar
+              </InoButton>
+              {showSnackbar && (
+                <InoSnackbar
+                  action-text="Some Action"
+                  timeout={5000}
+                  type="info"
+                  stay-visible-on-hover="false"
+                  onHideEl={() => {
+                    setSnackbar(false);
+                  }}
+                >
+                  This is a message
+                </InoSnackbar>
+              )}
+            </div>
+            <div slot="footer" className={styles.footer}>
+              <div className={styles.component}>Snackbar</div>
+              <div className={styles.label}>Notification</div>
+            </div>
+          </InoCard>
+        </div>
+        <div className={styles.input}>
+          <InoCard disableElevation={true}>
```

Comment:
I would suggest extracting the card in its own component, so we dont have to write `disableElevation={true}` on every card.